strings.txt

```diff
@@ -16217,10 +16217,12 @@
 
   [editor_storey_number]
     tags = ios, android
+    en = Storey number (maximum %1$@)
```

Comment:
?
> On Apr 21, 2016, at 3:57 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2947#discussion_r60574291>:
> 
> > @@ -16217,10 +16217,12 @@
> >  
> >    [editor_storey_number]
> >      tags = ios, android
> > +    en = Storey number (maximum %1$@)
> Building Levels
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2947/files/44b3f477a158544a38212dbfd95100ddbb3c305c#r60574291>